pkg/kube/cluster-init.sh

```diff
@@ -300,20 +300,34 @@ check_start_containerd() {
                 logmsg "Started k3s-containerd at pid:$containerd_pid"
         fi
         if [ -f /etc/external-boot-image.tar ]; then
+                # Give containerd a moment to start before importing
+                for _ in 1 2 3; do
```

Comment:
Is really 3 attempts each 1s enough? Have you measured the delay in different machines?

Reply:
@rene It should be enough, but to cover unknowns I have changed this to return if containerd is not running after one check.  This is ok as check_start_containerd() will be called again every 15 seconds and this will attempt again.